components/tabs/stories/tabs.stories.js

```diff
@@ -1,10 +1,24 @@
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { version } from "../package.json";
 import { TabsGroups } from "./tabs.test.js";
-import { Template } from "./template.js";
+import { Template, QuietGroup, OverflowGroup, VerticalGroup, CompactGroup } from "./template.js";
 
 /**
  * Tabs organize content into multiple sections and allow users to navigate between them. The content under the set of tabs should be related and form a coherent unit.
+ * 
+ * ## Use icons consistently
+ * 
+ * Icons are optional, but don’t mix the use of icons in tabs if they are utilized. Navigation controls require a clear spacial relationship to one another, and mixing the use of icons can dramatically impact the visual balance and presence for each tab item.
```

Comment:
Should this "spacial" be spatial? (Or maybe special?)

Reply:
Who would I ask about that? This is copied right from the guidance site: https://spectrum.adobe.com/page/tabs/#Use-icons-consistently

Doing a quick google, I think "spatial" _was_ the correct way to spell this, but now "spacial" is sort of accepted.